docs/source/config.json

```diff
@@ -2,130 +2,16 @@
   "title": "Self-Hosted Router",
```

Comment:
I'm a little confused about the presence of this file since it powered the sidebar on the old docs--do we need it at all?

Reply:
Summary of offline our conversation: CI currently uses config.json to check for the existence of its listed files, so needed to remove the outdated files from it. Still keeping the file around as a starting point to discuss how we can/should create a similar "manifest" of all router docs files.